controller/config.go

```diff
@@ -600,6 +618,19 @@ func (c Config) AllowModelAccess() bool {
 	return value
 }
 
+// ModelLogfileMaxBackups is the number of old model log files to keep (compressed).
+func (c Config) ModelLogfileMaxBackups() int {
+	return c.intOrDefault(ModelLogfileMaxBackups, DefaultModelLogfileMaxBackups)
+}
+
+// ModelLogfileMaxSizeMB is the maximum size of the log file written out by the
+// controller on behalf of workers running for a model.
+func (c Config) ModelLogfileMaxSizeMB() int {
+	// Value has already been validated.
+	val, _ := utils.ParseSize(c.mustString(ModelLogfileMaxSize))
```

Comment:
Does this need defaulting?

Reply:
No, because the configuration validation process has pushed though the schema checker, which adds a default value if one isn't set.